app/services/user-settings.js

```diff
@@ -0,0 +1,67 @@
+import Object from '@ember/object';
+import Service from '@ember/service';
+import {getObject, setObject} from '../utils/storage';
+
+const SETTINGS_KEY = 'user_settings';
+const TOKENS_KEY = 'user_tokens';
+const PROPERTIES = ['githubTokenModalSeen'];
+
+// Use github_com instead of github.com for localstorage because of ember `set`
+// and `get` internals
+const userSettings = Service.extend({
+  init() {
+    this._super(...arguments);
+    this.tokens = Object.create({
```

Comment:
A little confused as to why you're using `Object.create()` here instead of just:

```javascript
this.tokens = {
  'github_com': null,
}
```

Reply:
It lives in ember service, the property is exposed. Ember object has addtional get and set method. Doing something like userSettings.get('tokens').set('gitlab_cern_ch') is common in ember.

Reply:
See import statement.

Reply:
See import statement.

Reply:
I try to thin as ember developer. And many resource use get and set. If you think it is not commonin ember. I will change.

Reply:
Yep I see it now... Wish Ember had named that differently so it could be differentiated from JavaScript's built-in Object class...

Reply:
Well, it default export. My bad, should've name it `EmberObject`.
Will rename it in the next push.